main.js

```diff
@@ -0,0 +1,109 @@
+
+// var pegs = 3;
+// var discs = 5;
+
+const board = [];
+
+function startGame(board) {
+this.board = [
+  [5, 4, 3, 2, 1],
+  [],
+  []
+  ];
+console.log(' ');
+console.log('TOWERS OF HANOI');
+console.log(' ');
+console.log('--- ' + this.board[0].join(' '));
+console.log('--- ');
+console.log('--- ');
+console.log(' ');
+};
+
+
+
+
+function moveDisc(a, b) { 
+  a -= 1;
```

Comment:
use descriptive naming so it helps people who are reading code - a doesn't tell me what is in it